test-network/scripts/createChannel.sh

```diff
@@ -94,6 +94,10 @@ setAnchorPeer() {
 }
 
 
+## User attempts to use BFT orderer in Fabric network with CA
+if [ $BFT -eq 1 ] && [ -d "organizations/fabric-ca/ordererOrg/msp" ]; then
+  fatalln "Fabric network seems to be using CA. This sample does not yet support the use of consensus type BFT and CA together."
+fi
```

Comment:
This test should be done after parsing the command line arguments so that it is also triggered if one simply does: "./network.sh up -ca -bft"